packages/python/plotly/test_requirements/requirements_39_optional.txt

```diff
@@ -19,3 +19,4 @@ matplotlib==2.2.3
 scikit-image==0.18.1
 psutil==5.7.0
 kaleido
+orjson==3.8.12
```

Comment:
I added `orjson` only to py3.7 and py3.9 optional tests, so that we'd run a bunch of the other tests using `json`. Recent `orjson` doesn't support py3.6 anyway.